webapp/TargetedMS/js/QCPlotHelperBase.js

```diff
@@ -645,7 +647,8 @@ Ext4.define("LABKEY.targetedms.QCPlotHelperBase", {
                 },
                 yLeft: {
                     value: this.getYScaleLabel(plotType, trendLineProps.valueConversion, metricProps.series1Label),
-                    color: this.isMultiSeries() ? this.getColorRange()[0] : undefined
+                    color: this.isMultiSeries() ? this.getColorRange()[0] : undefined,
+                    position: this.getYAxisPosition(precursorInfo, leftMargin)
```

Comment:
If you move `this.getYAxisLeftMarginOffset(precursorInfo)` to a variable above, then `getYAxisPosition` can probably go away and just be inlined here.
```suggestion
                    position: marginOffset > 0 ? leftMargin + marginOffset - 15 : undefined
```

Reply:
done